File: apps/android/app/src/main/java/com/example/assistant/CoreBridge.kt
package com.example.assistant

// UniFFI bridge placeholder. TODO: replace with generated bindings + JNI wiring.
class CoreBridge {
    fun ping(): String = "core-bridge-stub"

    fun chatRequest(messagesJson: String, requireConfirmation: Boolean): String {
        return "{\"status\":\"stub\",\"messages\":$messagesJson,\"requireConfirmation\":$requireConfirmation}"
    }
}

File: apps/ios/AssistantApp/CoreBridge.swift
import Foundation

// UniFFI bridge placeholder. TODO: replace with generated bindings and Rust FFI wiring.
struct CoreBridge {
    func ping() -> String {
        "core-bridge-stub"
    }

    func chatRequest(prompt: String, requireConfirmation: Bool) -> String {
        let mode = requireConfirmation ? "RequireConfirmation" : "BestEffort"
        return "iOS bridge stub -> mode=\(mode), prompt=\(prompt)"
    }
}
